src/main/java/ch/njol/skript/Skript.java

```diff
@@ -463,7 +475,7 @@ public void run() {
 								final String c = e.getName().replace('/', '.').substring(0, e.getName().length() - ".class".length());
 								try {
 									final Class<?> hook = Class.forName(c, true, getClassLoader());
-									if (hook != null && Hook.class.isAssignableFrom(hook) && !hook.isInterface() && Hook.class != hook) {
+									if (hook != null && Hook.class.isAssignableFrom(hook) && !hook.isInterface() && Hook.class != hook && !disabledHookRegistrations.contains(hook)) {
```

Comment:
I think it would make more sense to have a public method to check if a hook is enabled:
```
public static boolean isHookEnabled(Class<? extends Hook<?>> hook) {
    return !disabledHookRegistrations.contains(hook);
}
```
then use it here. That way the intent of the check if more clear, and there's an API for add-ons to check if a hook is enabled or not.